packages/packages.json

```diff
@@ -1,62 +1,62 @@
+        "agent/valory/counter/0.1.0": "bafybeia2ljyztywxuqibcxowqwtybnn647cvwvxcm6jmye5un64tcrllxm",
+        "agent/valory/counter_client/0.1.0": "bafybeibjaof2toe4csn4bt2xaglnrgsu4tzbeczmxaer3w7s3qvkujj3qa",
+        "agent/valory/register_reset/0.1.0": "bafybeif3y4asf54j6d27itckqc4aqfacs6a6lgbir5o4ikejx7pvfi6n3e",
+        "agent/valory/register_termination/0.1.0": "bafybeieaqifsjxmwf4llgpph6bzemqytji57y5zqkfwglbdcehfipzuheu",
+        "agent/valory/registration_start_up/0.1.0": "bafybeihfpxjhgmi266cqst54d64tnskmew6rmrcry5d5otml5wzui7uy7i",
+        "agent/valory/test_abci/0.1.0": "bafybeihouc3dswh2ecjne6rsvu54vxp4ztkyuxnnktlylpkvnvlzke7ine",
+        "agent/valory/register_reset_recovery/0.1.0": "bafybeih577meamtzilyjazaatmqqvfcypf6d6hpxkdge2cbt5lkmcfl3jm",
+        "agent/valory/offend_slash/0.1.0": "bafybeihrftyaidfh3k7tjnjylt2k4ucjkziyk5sgbeevfwfvqexwyzuf7m",
+        "agent/valory/solana_transfer_agent/0.1.0": "bafybeigbjovmgodubzaif6qwu3awbiemw25pk5chfniiu6dkqy2pyf4jgy",
+        "service/valory/counter/0.1.0": "bafybeibd5jjoxuuuawca2jcrwijrcsflqknbx6bgaqaehvabxzs3kfxdma",
+        "service/valory/register_reset/0.1.0": "bafybeibj3qzduzurusnt55lkyjy657qinl725beb2l2julzrtjw6edy2si"
     },
     "third_party": {
-        "protocol/open_aea/signing/1.0.0": "bafybeihv62fim3wl2bayavfcg3u5e5cxu3b7brtu4cn5xoxd6lqwachasi",
-        "protocol/valory/acn/1.1.0": "bafybeidluaoeakae3exseupaea4i3yvvk5vivyt227xshjlffywwxzcxqe",
-        "protocol/valory/http/1.0.0": "bafybeifugzl63kfdmwrxwphrnrhj7bn6iruxieme3a4ntzejf6kmtuwmae",
-        "protocol/valory/ledger_api/1.0.0": "bafybeihdk6psr4guxmbcrc26jr2cbgzpd5aljkqvpwo64bvaz7tdti2oni",
-        "protocol/valory/contract_api/1.0.0": "bafybeidgu7o5llh26xp3u3ebq3yluull5lupiyeu6iooi2xyymdrgnzq5i",
-        "connection/valory/http_client/0.23.0": "bafybeihi772xgzpqeipp3fhmvpct4y6e6tpjp4sogwqrnf3wqspgeilg4u",
-        "connection/valory/ledger/0.19.0": "bafybeigntoericenpzvwejqfuc3kqzo2pscs76qoygg5dbj6f4zxusru5e",
-        "connection/valory/p2p_libp2p_client/0.1.0": "bafybeid3xg5k2ol5adflqloy75ibgljmol6xsvzvezebsg7oudxeeolz7e"
+        "protocol/open_aea/signing/1.0.0": "bafybeibnvolculfakdhx2ah7vuoubgnuf2dhkv44baxw6y7j2zwdrptdpe",
+        "protocol/valory/acn/1.1.0": "bafybeib5o5ey563riygpkzgkfab3dtvz5d3qpcdbnfuffzpac6uuu7ppmy",
+        "protocol/valory/http/1.0.0": "bafybeiex7nd5ehkocamuael2qkeq3r73bfa7si6ygmy2tgmztkb443ul4u",
+        "protocol/valory/ledger_api/1.0.0": "bafybeiglagji3abfdnciqvakdr7hm2owjkasesuhksi4m46lbqkg3525le",
+        "protocol/valory/contract_api/1.0.0": "bafybeib6pi4cban4pt7vu2avbgeh5seic6r7lgio73tqguh2uv564qunqe",
+        "connection/valory/http_client/0.23.0": "bafybeia3s5lbmcxjj5zo6y2qg3aow4nralbpchgacnkxn6h3ph3dgzyvem",
+        "connection/valory/ledger/0.19.0": "bafybeibjlqbfs4zaa6woefuxcliugljb7cgo2iyy2ilhc4jsyoijomqbkq",
+        "connection/valory/p2p_libp2p_client/0.1.0": "bafybeidkdop5q4brjezeu3edpfdud3rbwmpv33tqql2lqydfvbfztvtkbq"
```

Comment:
Why did you perform these changes in the third-party packages?

Reply:
to fix this CI error https://github.com/valory-xyz/open-autonomy/actions/runs/12585693644/job/35078013489?pr=2288

Reply:
Afaict you did right re-generating the protocols, but the third-party packages should match the latest ones. Therefore, I think that the only one that needs to be changed is [p2p_libp2p_client](https://github.com/valory-xyz/open-aea/blob/2ad8211f758508a968a1d0ae7da0feb3cf04395a/packages/packages.json#L20).

Reply:
The third-party packages should match the latest ones. Therefore, I think that the only one that needs to be changed is [p2p_libp2p_client](https://github.com/valory-xyz/open-aea/blob/2ad8211f758508a968a1d0ae7da0feb3cf04395a/packages/packages.json#L20).